src/sinks/splunk_hec/logs/sink.rs

```diff
@@ -124,6 +127,7 @@ pub fn process_log(
     index: Option<&Template>,
     host_key: &str,
     indexed_fields: &[String],
+    timestamp_nanos_key: Option<&String>,
```

Comment:
Can this be `Option<&str>`? `&String` is usually discouraged as being pointless.

Reply:
Yes, I updated the PR accordingly.